tests/test_rewind.py

```diff
@@ -98,7 +105,7 @@ def test__check_timeline_and_lsn(self, mock_check_leader_is_not_in_recovery):
         with patch('psycopg2.connect', Mock(side_effect=Exception)):
             self.assertFalse(self.r.rewind_or_reinitialize_needed_and_possible(self.leader))
         self.r.trigger_check_diverged_lsn()
-        with patch.object(MockCursor, 'fetchone', Mock(side_effect=[('', 2, '0/0'), ('', b'3\t0/40159C0\tn\n')])):
+        with patch.object(MockCursor, 'fetchone', Mock(side_effect=[('', 3, '0/0'), ('', b'3\t0/40159C0\tn\n')])):
```

Comment:
```suggestion
        with patch.object(MockCursor, 'fetchone', Mock(side_effect=[('', 3, '0/0'), ('', b'4\t0/40159C0\tn\n')])):
```
not sure what the 3 stands for, but as some other numbers are raised, I thought, I'd question this one.

Reply:
This is just a specially crafted mock data, the purpose of it to bring the system into a state that it will go through certain code-paths.